python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -979,6 +980,83 @@ def convert_to_cudf(cp_arrays):
 
         return ddf
 
+    def decompress_to_edgelist(
```

Comment:
I only see this on the MG Graph API, shouldn't this also be added to the SG Graph API?

Reply:
I pushed the SG API